bin/cron/deliver_poste_messages

```diff
@@ -245,20 +245,37 @@ class Deliverer
     @templates[name] = Poste::Template.new IO.read(path), engine
   end
 
-  def parse_addresses(string)
-    string.split(', ').map do |user_string|
-      if user_string.include? '<'
-        user_string =~ /^([^<]*)<([^>]*)>$/
-        {name: $1.strip, email: $2}
-      else
-        {name: nil, email: user_string}
-      end
+  def parse_address_string(string)
+    # A comma without surrounding double-quotes isn't necessarily bad, since it
+    # might be like this:
+    #   friendly name, phd <email@code.org>
+    #
+    # Though in such a case, it should really be provided like this:
+    #   "friendly name, phd", <email@code.org>
+    #
+    # But we are alerting just in case a sender has attempted to provide multiple
+    # email addresses like this:
+    #   friendly name <email@code.org>, second person <email2@code.org>
+    #
+    # Because we deprecated support for this pattern, believing that we've never
+    # actually used it.
+    if /,(?=(?:[^\"]*\"[^\"]*\")*[^\"]*$)/ =~ string
```

Comment:
In the change from your last approach we lost the comment explaining where you got this regex and why it works.  Can we bring that back?